component/http/middleware/middleware.go

```diff
@@ -487,38 +497,14 @@ func logRequestResponse(corID string, w *responseWriter, r *http.Request) {
 	}
 
 	info := map[string]interface{}{
-		"request": map[string]interface{}{
-			"remote-address": remoteAddr,
-			"method":         r.Method,
-			"url":            r.URL,
-			"proto":          r.Proto,
-			"status":         w.Status(),
-			"referer":        r.Referer(),
-			"user-agent":     r.UserAgent(),
-			correlation.ID:   corID,
-		},
-	}
-	log.Sub(info).Debug()
-}
-
-func getOrSetCorrelationID(h http.Header) string {
-	cor, ok := h[correlation.HeaderID]
-	if !ok {
-		corID := uuid.New().String()
-		h.Set(correlation.HeaderID, corID)
-		return corID
-	}
-	if len(cor) == 0 {
-		corID := uuid.New().String()
-		h.Set(correlation.HeaderID, corID)
-		return corID
-	}
-	if cor[0] == "" {
-		corID := uuid.New().String()
-		h.Set(correlation.HeaderID, corID)
-		return corID
-	}
-	return cor[0]
+		correlation.ID:   corID,
+		"method":         r.Method,
+		"url":            r.URL,
+		"status":         w.Status(),
+		"remote-address": remoteAddr,
+		"proto":          r.Proto,
+	}
+	log.FromContext(r.Context()).Sub(info).Debug()
```

Comment:
I know this was already present before the code change, but it will print an empty message, is it desirable?

Reply:
The message will not be empty. It will contain all the info data. Should I add a message?

Reply:
It might make it easier to understand what the log message is about; but no worries, this is not important.